### PR TITLE
OpenStack: Better Docs Organization and Linking

### PR DESCRIPTION
**Pull request by iamemilio**
Adds a table of contents, sorts sections by type of activity, created a whole new doc for Kuryr related info, and links important reference docs.

### CODE REVIEW

**Comment by iamemilio** [comment]
@maxwelldb

File: docs/user/openstack/README.md [hunk 1]
```diff
@@ -1,51 +1,54 @@
...
+    - [Red Hat Enterprise Linux CoreOS (RHCOS)](#red-hat-enterprise-linux-coreos-rhcos)
+    - [Neutron Public Network](#neutron-public-network)
+  - [OpenStack Credentials](#openstack-credentials)
+  - [Standalone Single-Node Development Environment](#standalone-single-node-development-environment)
+  - [Running The Installer](#running-the-installer)
+    - [Initial Setup](#initial-setup)
+    - [API Access](#api-access)
+      - [Using Floating IPs](#using-floating-ips)
+      - [Without Floating IPs](#without-floating-ips)
+    - [Running a Delpoyment](#running-a-delpoyment)
```
Comment (maxwelldb) [inline docs/user/openstack/README.md]: Typo right here, @iamemilio

Comment (Fedosin) [reply docs/user/openstack/README.md]: this is how we delpoy it!

File: docs/user/openstack/README.md [hunk 2]
```diff
@@ -234,72 +214,37 @@ Alternatively, if you don't control the DNS, you can add the hostnames in `/etc/
...
-* There is a known limitation of Octavia not supporting listeners on different protocols (e.g., UDP and TCP) on the same port. Thus services exposing the same port for different protocols are not supported.
-* Due to the above UDP limitations of Octavia, Kuryr is forcing pods to use TCP for DNS resolution (`use-vc` option at `resolv.conf`). This may be a problem for pods running Go applications compiled with `CGO_DEBUG` flag disabled as that forces to use the `go` resolver that is only using UDP and is not considering the `use-vc` option added by Kuryr to the `resolv.conf`. This is a problem also for musl-based containers as it's resolver does not support `use-vc` option. This would include e.g., images build from `alpine`.
+### Running a Delpoyment
 
-## Running The Installer
-
-Finally, you can run the installer with the following command:
+The following instructions are for how to run a default cluster. If your want to customize your install, see the [customizing docs](customization.md). For information on running the insaller with kuryr, see the [kuryr docs](kuryr.md).
```
Comment (Fedosin) [inline docs/user/openstack/README.md]: typo: installer

Comment (Fedosin) [inline docs/user/openstack/README.md]: it's better to capitalize it: Kuryr

**Comment by Fedosin** [comment]
I'm happy to approve this pr when the typos are fixed. Thanks @iamemilio !

File: docs/user/openstack/README.md (hunk 1 above)
Comment (iamemilio) [reply docs/user/openstack/README.md]: LOL it legit took me 10 minutes to realize what the typo was

**Pull request merged**